.github/workflows/delete-review-app.yml

```
@@ -0,0 +1,85 @@
+        with:
+          script: |
+            const comment = await github.rest.issues.createComment({
+              issue_number: process.env.PR_NUMBER,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: '🗑️ Starting app deletion...'
+            });
+            return { commentId: comment.data.id };
+
+      - name: Delete Review App
+        uses: ./.github/actions/delete-control-plane-app
```

actionlint (1.7.4)

missing input "app_name" which is required by action "Delete Control Plane App" defined at "./.github/actions/delete-control-plane-app". all required inputs are "app_name", "org"

(action)